LLama/Common/FixedSizeQueue.cs

```diff
@@ -83,17 +125,40 @@ IEnumerator IEnumerable.GetEnumerator()
             return GetEnumerator();
         }
 
+        /// <summary>
+        /// Returns up to <paramref name="count"/> of the most recent items as a contiguous span.
+        /// </summary>
         internal ReadOnlySpan<T> AsSpan(int count)
```

Comment:
`AsSpan` doesn't seem to be used anywhere, so this could be removed if you want. (I have no objection if you would prefer to keep it).